internal/prometheus/prometheus.go

```diff
@@ -36,26 +36,15 @@ type Provider struct {
 
 type loggerFunc func(attrs map[string]interface{}, f string, a ...interface{})
 
-// New returns a new Provider object.
-func New(inner metrics.Metrics, logger loggerFunc) *Provider {
+// NewWithHTTPRequestBuckets returns a new Provider object.
+func NewWithHTTPRequestBuckets(inner metrics.Metrics, logger loggerFunc, httpRequestBuckets []float64) *Provider {
```

Comment:
I was suggesting we keep `New` as is with the default buckets and add a new function like `func (p *Provider) WithHTTPRequestHistogramBuckets(buckets []float64) *Provider`. But now that I look at the prom lib, you won't be able to override the buckets once initialized. Your original change w/o the name change seems fine then. Sorry about that.